server/models/notification.js

```diff
@@ -0,0 +1,39 @@
+const mongoose = require("mongoose");
+
+const notificationSchema = new mongoose.Schema({
+  userId: {
+    type: mongoose.Schema.Types.ObjectId,
+    required: true,
+    ref: "User",
+  },
```

Comment:
I think it's fine to just have one user in this notification. We'll never have an instance of fetching another users notifications.

Reply:
My thought was an Id of someone who sent the notification and one who receives it. because I saw on the mock, a notification that said "so and so requested for your service" If am right my opinion was we need both